cli/cmd/vuln_container.go

```diff
@@ -754,3 +804,93 @@ func filterSeverity(severity string, threshold string) bool {
 func vulFiltersEnabled() bool {
 	return vulCmdState.Severity != "" || vulCmdState.Fixable
 }
+
+func getContainerRegistries() ([]string, error) {
+	var (
+		registries            = make([]string, 0)
+		regsIntegrations, err = cli.LwApi.Integrations.ListContainerRegistries()
+	)
+	if err != nil {
+		return registries, errors.Wrap(err, "unable to get container registry integrations")
+	}
+
+	for _, i := range regsIntegrations.Data {
+		// avoid adding empty registries coming from the new local_scanner
+		if i.Data.RegistryDomain == "" {
+			continue
+		}
+
+		// avoid adding duplicate registries
+		if array.ContainsStr(registries, i.Data.RegistryDomain) {
+			continue
+		}
```

Comment:
Could reduce this?

`if i.Data.RegistryDomain == "" || array.ContainsStr(registries, i.Data.RegistryDomain){`